qemu/tests/cfg/numa_negative.cfg

```diff
@@ -0,0 +1,39 @@
+- numa_negative:
+    type = numa_negative
+    kill_vm = yes
+    kill_vm_gracefully = no
+    backend_mem = memory-backend-ram
+    use_mem = no
+    start_vm = no
+    smp = 2
+    vcpu_maxcpus = ${smp}
+    mem = 2048M
+    mem_devs = 'mem0 mem1'
+    size_mem = 1024M
+    guest_numa_nodes = 'node0 node1'
+    numa_memdev_node0 = mem-mem0
+    numa_memdev_node1 = mem-mem1
+    numa_cpus_node0 = '0'
+    numa_cpus_node1 = '1'
+    variants negative_type:
+        - threads_test:
+            vcpu_sockets = 1
+            vcpu_dies = 1
+            vcpu_cores = 1
+            vcpu_threads = 2
```

Comment:
@MiriamDeng  For ppc, core-id must be the multiple of `vcpu_threads`,  one core-id can represent `vcpu_threads` cpus.
So, seems it is impossible to test 'threads in different node' with ppc, isn't it ?

Reply:
Yes, it is, I thought you metioned this test case numa_negative.threads_test.

Reply:
@MiriamDeng  Thanks for your info, could you update the test case for scenario "Boot up guest with "-numa node" set up the cpus in one node to some unsupported value like threads in different node:" first ?
For ppc, which one is the expected result ?
1. This is a x86 only scenario
2. The expected result is launch qemu failed.
Thanks!

Reply:
@MiriamDeng  Updated as we discussed, use different expected result for ppc, could you help review again, thanks!